packages/engine/src/actions/callMethod.test.js

```diff
@@ -148,7 +160,11 @@ test('CallMethod method return a promise', async () => {
                 {
                   id: 'a',
                   type: 'CallMethod',
-                  params: { blockId: 'textInput', method: 'blockMethod', args: ['arg'] },
+                  params: {
+                    blockId: 'block:root:textInput:0',
```

Comment:
This should only be `blockId: textInput ` as that is what the user would have in their config